eq-author/src/App/history/deleteHistoryNoteMutation.graphql

```diff
@@ -0,0 +1,8 @@
+#import "./historyNoteFragment.graphql"
+
+mutation DeleteHistoryNote($input: deleteHistoryNoteInput!) {
+  deleteHistoryNote(input: $input) {
+    id
```

Comment:
Why is `id` needed here? I think we are getting it of the `HistoryNoteInfo` fragment.